src/components/Feedback.js

```diff
@@ -0,0 +1,91 @@
+import { h } from 'preact'
+import { useState, useRef, useEffect } from 'preact/hooks'
+import { useI18n, useSoftkey, useNavigation } from 'hooks'
+import { sendFeedback } from 'utils'
+
+export const Feedback = ({ close }) => {
+  const containerRef = useRef()
+  const i18n = useI18n()
+  const [message, setMessage] = useState()
+  const [validInput, setValidInput] = useState(null)
+  const [, setNavigation, getCurrent] = useNavigation('Feedback', containerRef, containerRef, 'y')
+
+  const items = [
+    { text: `<a data-selectable>${i18n('feedback-terms-of-survey')}</a>`, link: 'https://wikimediafoundation.org/' }, // TODO: link not provided yet
+    { text: `<a data-selectable>${i18n('feedback-privacy-policy')}</a>`, link: 'https://foundation.m.wikimedia.org/wiki/Privacy_policy' },
+    { text: `<a data-selectable>${i18n('feedback-terms-of-use')}</a>`, link: 'https://foundation.m.wikimedia.org/wiki/Terms_of_Use/en' }
+  ]
+  const hyperlinks = items.map(i => i.text)
+
+  const onKeyRight = () => {
+    if (message) {
+      sendFeedback(message)
+      setValidInput(true)
+    } else {
+      setValidInput(false)
+    }
+  }
+
+  const onKeyBackspace = () => {
+    if (message && getCurrent().type === 'TEXTAREA') {
+      setMessage(message.slice(0, -1))
```

Comment:
This is the default behavior of the backspace key. Do you know why you have to do it manually? I don't think we do in the search input.

Reply:
The behavior I see without this logic doesn't handle deleting characters, and currently we're doing the same in the `onKeyBackspace` for the input in `Language` and `ArticleLanguage` ([see here)](https://github.com/wikimedia/wikipedia-kaios/blob/master/src/components/Language.js#L29). 

This was introduced when we implemented the backspace key feature to navigate the app. The reason we don't have this in `Search` is because we didn't need to alter the behavior of the backspace key on that page, there we needed the backspace key to exit the app which is the original behavior.

But perhaps there's a simpler way to do this? 🤔

Reply:
We can have the useSoftkey like

```js
useSoftkey('Feedback', {
    right: i18n('send'),
    onKeyRight,
    left: i18n('softkey-close'),
    onKeyLeft: close,
    onKeyBackspace: (message && getCurrent().type === 'TEXTAREA') || close,
    onKeyCenter
}, [message, current])
```

not simpler nor pretty, but we don't need to define slice event